pkg/services/sqlstore/migrations/live.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+import (
+	. "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func addLiveMigrations(mg *Migrator) {
+	liveChannel := Table{
+		Name: "live_channel",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, Nullable: false, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "org_id", Type: DB_BigInt, Nullable: false},
+			{Name: "channel", Type: DB_NVarchar, Length: 189, Nullable: false},
```

Comment:
Do we need id?  can we do a composite primary key on `org_id+channel`?

Reply:
We are not using it, but let me check whether it's possible with current sql migrator

Reply:
Would advise using Id as primary key to follow the convention of all (most) of the other tables. org_ud+channel can be a unique index.

Reply:
Checked it out - it's possible to use composite primary key, but it results into raw SQL for most of queries (select, insert) since xorm builds queries with id column by default. Agree with @marefr - it's simpler to follow common approach, the overhead should be tolerable.